docs/source/release_notes/release-notes.rst

```diff
@@ -42,23 +44,71 @@ develop
     *    - Type
          - Change
 
+    *    -
+         -
+
+
+.. <<<<------------------------------------------------------------------------------------------------------------->>>>
+
+======
+0.46.0
+======
+
+29 June 2017
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+
+    *    - Type
+         - Change
+
+    *    - |userbreak| |improved|
```

Comment:
This is _both_ a User and a Dev break (it's Dev because of the first part you mention, on the TMs.create() - and User because of the options may not be specified stuff)